PySDM/backends/numba.py

```diff
@@ -7,42 +7,39 @@
 from PySDM.backends.impl_numba.methods.condensation_methods import CondensationMethods
 from PySDM.backends.impl_numba.methods.displacement_methods import DisplacementMethods
 from PySDM.backends.impl_numba.methods.freezing_methods import FreezingMethods
-from PySDM.backends.impl_numba.methods.index_methods import IndexMethods
 from PySDM.backends.impl_numba.methods.moments_methods import MomentsMethods
-from PySDM.backends.impl_numba.methods.pair_methods import PairMethods
 from PySDM.backends.impl_numba.methods.physics_methods import PhysicsMethods
 from PySDM.backends.impl_numba.methods.terminal_velocity_methods import (
     TerminalVelocityMethods,
 )
-from PySDM.backends.impl_numba.random import Random as ImportedRandom
-from PySDM.backends.impl_numba.storage import Storage as ImportedStorage
 from PySDM.formulae import Formulae
+from PySDM.storages.holders.numba import NumbaStorageHolder
+from PySDM.storages.numba.backend.index import IndexBackend
+from PySDM.storages.numba.backend.pair import PairBackend
 
 
 class Numba(  # pylint: disable=too-many-ancestors,duplicate-code
     CollisionsMethods,
-    PairMethods,
-    IndexMethods,
+    PairBackend,
+    IndexBackend,
     PhysicsMethods,
     CondensationMethods,
     ChemistryMethods,
     MomentsMethods,
     FreezingMethods,
     DisplacementMethods,
     TerminalVelocityMethods,
+    NumbaStorageHolder,
 ):
-    Storage = ImportedStorage
-    Random = ImportedRandom
-
     default_croupier = "local"
 
+    # TODO #1185  https://github.com/pylint-dev/pylint/issues/9225
+    # pylint: disable=super-init-not-called
```

Comment:
This pragma should not be needed here https://github.com/open-atmos/PySDM/issues/1185